docs/config/aws_template.md

```diff
@@ -0,0 +1,431 @@
+# Deploying to AWS
+
+!!! danger "AWS Admin Privileges Required"
+
+    This deployment can only be run by AWS administrators, so we assume the user has a basic understanding of code
+    development, Docker containers, and the AWS cloud.
+
+## Overview
+
+The goal of this guide is to help you create a functioning pipeline in AWS that allows you to process data on-the-fly as
+it enters S3 buckets (Ingest applications), or on a regular cron schedule (VAP applications), and store the processed
+results in an output S3 bucket in a structured format.
+
+=== "Deployment result"
+
+    Ingest and VAP pipelines can be set up to run on S3 event triggers or on a cron schedule.
+    
+    ![User friendly overview](../figures/aws/user_friendly_overview.png)
+
+=== "Stack resources"
+
+    The following resources will be used or setup during the AWS deployment. 
+    
+    ![AWS Template](../figures/aws/stack_resources.png)
+
+=== "CodeBuild steps"
+
+    The following image shows how code changes are deployed to AWS and the steps that make up the AWS CodeBuild process
+    to update or create lambda functions and containers for each pipeline.
+
+    ![Image](../figures/aws/code_build.png)
+
+## Prerequisites
+
+### Create code repos
+
+Make sure that you have two repositories in your GitHub organization/account created from the following templates:
+
+* **<https://github.com/tsdat/pipeline-template>**
+* **<https://github.com/tsdat/aws-template>**
+
+If you are using an existing `pipeline-template` repository, make sure that the `requirements.txt` file specifies a
+`tsdat` version of at least `tsdat==0.7.1`. The AWS build will not work with earlier versions of `tsdat`.
+
+Clone these repos to the same parent folder on your computer.
+
+???+ warning "Warning: Windows users"
+
+    If you are using WSL on Windows make sure you run the `git clone` command from a WSL terminal.
```

Comment:
Add reason why?

Reply:
She added it because of the file encoding problem I originally ran into with \r, I believe

Reply:
Ah, gotcha; to prevent changing the line endings then. Should we keep the warning for this, then?

Reply:
Yes.  We should clarify the text to:
`If you are using WSL on Windows, make sure you run the `git clone` command from a WSL terminal to prevent git from converting all the file line endings to CRLF.  If your files have CRLF line endings, it will cause the AWS pipeline to crash.`

Reply:
Added this warning back:

<img width="1049" alt="image" src="https://github.com/tsdat/tsdat/assets/24307537/e117c81a-4811-4d76-82ee-0f85ba03e96a">